src/main/java/com/ldtteam/structurize/placement/handlers/placement/PlacementHandlers.java

```diff
@@ -687,6 +688,10 @@ public List<ItemStack> getRequiredItems(
           @Nullable final CompoundTag tileEntityData,
           final boolean complete)
         {
+            if (complete)
+            {
+                return Collections.singletonList(new ItemStack(blockState.getBlock()));
+            }
             return new ArrayList<>();
```

Comment:
shouldn't this always be the block?

Reply:
No, this is when the world block matches